### ISSUE TITLE
Editorial Workflow entries published from editor view show wrong content.

### ISSUE THREAD

**Issue by tech4him1** [netlify/netlify-cms#1229]
**- Do you want to request a *feature* or report a *bug*?**

bug

**- What is the current behavior?**

If an entry is modified, saved, and published from the editor screen, the markdown widget reverts back to the old content when "Publish" is clicked. This doesn't seem to happen with any other widgets, the preview continues showing the new content, and it resolves if you go out the the collection view and back in.

**- If the current behavior is a bug, please provide the steps to reproduce.**
1. Edit the markdown for an existing entry.
2. Click "Save".
3. Change status to "Ready" (in the editor, not on the WorkFlow page).
4. Click "Publish" (in the editor, not on the WorkFlow page).
5. Not that the markdown editor does not show your changes, although the preview does.

**- What is the expected behavior?**

An entry should always show the correct content.

**- Please mention your CMS, node.js, and operating system version.**

CMS v1.4.0

**- Please link or paste your `config.yml` below if applicable.**
```
publish_mode: editorial_workflow # optional, enables publishing workflow

collections: # A list of collections the CMS should be able to edit
  - name: "posts" # Used in routes, ie.: /admin/collections/:slug/edit
    label: "Posts" # Used in the UI
    label_singular: "Post" # Used in the UI, ie: "New Post"
    description: >
      The description is a great place for tone setting, high level information, and editing
      guidelines that are specific to a collection.
    folder: "_posts"
    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
    create: true # Allow users to create new documents in this collection
    fields: # The fields each document in this collection have
      - {label: "Title", name: "title", widget: "string", tagname: "h1"}
      - {label: "Publish Date", name: "date", widget: "datetime", format: "YYYY-MM-DD hh:mma"}
      - {label: "Cover Image", name: "image", widget: "image", required: false, tagname: ""}
      - {label: "Body", name: "body", widget: "markdown"}
```

**Comment by erquhart** [comment]
Related to #754, common cause.

**Comment by brianlmacdonald** [comment]
@tech4him1 @erquhart  can I give this a shot?

**Comment by erquhart** [comment]
Fixed in #2129.
